README.md

```diff
@@ -41,9 +67,10 @@ Requirements for building
 A Makefile is provided for building tasks. The options are as follows
 
 Getting started is as simple as:
-```
-$ go get github.com/heptiolabs/gangway
-$ cd $GOPATH/src/github.com/heptiolabs/gangway
-$ make setup
-$ make
+
+```bash
+go get github.com/heptiolabs/gangway
```

Comment:
nit: We should add a `-u` to get latest each time.

Reply:
Fixed. Thanks Steve!